Xamarin.Essentials/Screenshot/Screenshot.ios.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using CoreTelephony;
+using Foundation;
+using UIKit;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Screenshot
+    {
+        internal static bool PlatformCanCapture => UIScreen.MainScreen != null;
+
+        static string GetTempFileName()
```

Comment:
Can we make use of the members on the FileSystem type? If not, it might be better to add something there for other APIs to use.

Reply:
No, the current FileSystem type does not provide the necessary functionality. Adding it does sound like a good idea. See the other post